server/ServerFSM.cpp

```diff
@@ -336,20 +336,6 @@ bool ServerFSM::EstablishPlayer(const PlayerConnectionPtr& player_connection,
 
         // inform player of host
         player_connection->SendMessage(HostIDMessage(m_server.m_networking.HostPlayerID()));
-
-        // send chat history
-        if (client_type == Networking::CLIENT_TYPE_HUMAN_MODERATOR ||
-            client_type == Networking::CLIENT_TYPE_HUMAN_OBSERVER ||
-            client_type == Networking::CLIENT_TYPE_HUMAN_PLAYER)
-        {
-            std::vector<std::reference_wrapper<const ChatHistoryEntity>> chat_history;
-            for (const auto& elem : m_server.GetChatHistory()) {
-                chat_history.push_back(std::cref(elem));
-            }
-            if (chat_history.size() > 0) {
-                player_connection->SendMessage(ChatHistoryMessage(chat_history));
-            }
-        }
     }
```

Comment:
What was wrong with sending history in other states? Now player could connect to `PlayingGame` state too.

Reply:
Remained from using player id, dropped this commit.